dkron/store.go

```diff
@@ -274,16 +274,24 @@ func (s *Store) GetLastExecutionGroup(jobName string) ([]*Execution, error) {
 	if err != nil {
 		return nil, err
 	}
-	if len(res) == 0{
+	if len(res) == 0 {
 		return []*Execution{}, nil
 	}
 
+	var lastEx Execution
 	var ex Execution
-	err = json.Unmarshal([]byte(res[len(res)-1].Value), &ex)
```

Comment:
this original line assumes that the LAST item of `res` is the LATEST Execution, the fact is that `res` is **unsorted**, so you have to iterate all Executions to find the LATEST one.